src/components/product/Pagination.component.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import styles from '@/components/product/Product.module.css';
+interface PropsPaginate {
+    productsPerPage: number;
+    currentPage: number;
+    totalProducts: any;
+    handlePageClick: any;
+    previousPage: () => void;
+    nextPage: () => void;
+}
+
+const Pagination: React.FC<PropsPaginate> = ({ productsPerPage, currentPage, totalProducts, handlePageClick, previousPage, nextPage }) => {
```

Comment:
move this component to the `pagination` folder and give props more generic names.
For example: `productsPerPage` -> `perPage`